tests/json.ml

```diff
@@ -5,13 +5,122 @@ let good = "
  { \"a\":\"b\" }
 "
 
-let _ =
-  Printf.printf "Parsing good JSON ... %!";
-  let _ = Jsonrpc.of_string good in
+let v1 = "{
```

Comment:
I like @lindig suggestion but I think this can be in the scope for a different PR where we update the whole test. As annoying to read and write as it is, it is anyway following the style of the other strings